.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Question
+about: Ask a question about ouch
+labels: question
```

Comment:
You should also throw some text to the question template description pointing to the [`Discussions`](https://github.com/ouch-org/ouch/discussions) tab, saying that questions can be asked there too.

Reply:
Should I redirect all questions to discussions tab and not have the template at all instead?

Reply:
I'm not sure, I guess we can leave both channels open so people can choose depending on what is the question about.